roles/openshift_facts/library/openshift_facts.py

```diff
@@ -56,6 +56,13 @@ def migrate_docker_facts(facts):
     if 'node' in facts and 'portal_net' in facts['node']:
         facts['docker']['hosted_registry_insecure'] = True
         facts['docker']['hosted_registry_network'] = facts['node'].pop('portal_net')
+
+    # log_options was originally meant to be a comma separated string, but
+    # we now prefer an actual list, with backward compatability:
+    if 'log_options' in facts['docker'] and \
+            isinstance(facts['docker']['log_options'], basestring):
+        facts['docker']['log_options'] = facts['docker']['log_options'].split(",")
```

Comment:
Are you testing immediately after the openshift_docker_facts role is run? the next time openshift_facts is run it would perform the migration. The easiest way to validate would be to add the following right after setting the value in openshift_docker_facts:
```
debug: var=openshift.docker.log_options
fail:
```